ros_gz_sim_demos/config/triggered_camera.yaml

```
@@ -0,0 +1,27 @@
+# triggered_cammera configuration.
+- ros_topic_name: "/camera"
+  gz_topic_name: "/camera"
+  ros_type_name: "sensor_msgs/msg/Image"
+  gz_type_name: "gz.msgs.Image"
+  subscriber_queue: 5
+  publisher_queue: 6
+  lazy: false
```

I don't think there's harm in making lazy=true.
```suggestion
  lazy: true
```

---

Changed in [3c5d49b](https://github.com/gazebosim/ros_gz/pull/645/commits/3c5d49b471c9a569691f3ffcf052ee437e23808d).